src/game/hud.cpp

```diff
@@ -169,7 +169,7 @@ namespace hud
     VAR(IDF_PERSIST, crosshairhitspeed, 0, 500, VAR_MAX);
     FVAR(IDF_PERSIST, crosshairblend, 0, 1, 1);
     FVAR(IDF_PERSIST, crosshairaccamt, 0, 0, 1);
-    VAR(IDF_PERSIST, crosshairflash, 0, 1, 1);
+    VAR(IDF_PERSIST, crosshairhealth, 0, 0b00, 0b11);
```

Comment:
Note that binary literals are a C++14 feature. I'm not sure if we can use those in Red Eclipse's codebase.